x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/MvExpandOperatorTests.java.orig

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.compute.operator;
+
+import org.elasticsearch.common.unit.ByteSizeValue;
+import org.elasticsearch.common.util.BigArrays;
+import org.elasticsearch.compute.data.Block;
+import org.elasticsearch.compute.data.BlockFactory;
+import org.elasticsearch.compute.data.ElementType;
+import org.elasticsearch.compute.data.IntBlock;
+import org.elasticsearch.compute.data.IntVector;
+import org.elasticsearch.compute.data.Page;
+
+import java.util.Iterator;
+import java.util.List;
+
+import static org.elasticsearch.compute.data.BasicBlockTests.randomBlock;
+import static org.elasticsearch.compute.data.BasicBlockTests.valuesAtPositions;
+import static org.elasticsearch.compute.data.BlockTestUtils.deepCopyOf;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+
+public class MvExpandOperatorTests extends OperatorTestCase {
```

Comment:
++. Thanks for adding this test.

Reply:
OoopS! That one was there before. thta's an `orig` from the patch. Ooops!

Reply:
Ah